autoscale/src/main/java/com/sequenceiq/periscope/controller/validation/AlertValidator.java

```diff
@@ -127,6 +128,25 @@ public void validateClusterLimit(AdjustmentType adjustmentType, Integer configur
         }
     }
 
+    public void validateScheduleWithStopStartAndDisable(Cluster cluster, DistroXAutoscaleClusterRequest autoscaleClusterRequest) {
+        if (!autoscaleClusterRequest.getTimeAlertRequests().isEmpty() &&
```

Comment:
I think it is possible for a DistroXAutoscaleClusterRequest to come in with just the boolean flags specified... so would need to check if there's already some ScheduleBasedPolicies registered for the cluster (not just new ones)

Reply:
Similar to the other method, I think this needs to check cluster for registered TimeAlertRequests. Also serves as a nice test case.

Reply:
Updated to check pre-existing and new schedule based policies for the cluster. Although, if a DistroXAutoscaleClusterRequest is sent with just boolean flags, it deletes all pre-existing alerts for the cluster, and doesn't end up creating any new policy.

https://github.com/hortonworks/cloudbreak/blob/be9073ed63283667ea901ce7610910f2f9d25f89/autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java#L165-L171